versioned_docs/version-2.2.0/helm-install.md

```diff
@@ -0,0 +1,40 @@
+---
+id: helm-install
```

Comment:
Some features were added to earlier versions. Have you double checked this with engineers?

Reply:
As mentioned in the PR description, I do think the doc sets for 2.2.0, 2.3.0 and 2.3.2 (and maybe more) have been overwritten by later versions unexpectedly during the website upgrade in last May and include many newly added features, e.g., TableView (introduced in 2.10.0) - I've removed the TableView related content though, but I believe there are much more, and challenging to check and clean one by one this way.

It would be more effective if there is a way to directly trace back the specific original doc versions; otherwise, those three doc sets have a huge gap in aligning with features. It's another thread, though. I discussed this with @urfreespace but the answer was negative.

So this PR aims to clean up 404 issues only. Despite this, I've checked with engineers about the availability of several features, e.g., schema, pulsar manager, security, etc.